src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+<!--
+  ~ Copyright (c) godot-rust; Bromeon and contributors.
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+NOTE: Keep track of where you install the Android Command Line Tools and JDK 17
+
+Exporting with gdext for godot requires some of the same pieces that are required for building godot from source.
+Specifically, the Android SDK Command Line Tools and JDK 17 as mentioned in Godot's documentation here: 
+https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#requirements
+
+Once you have those installed, you then need to follow godot's instructions for setting up the build system here:
+https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#setting-up-the-buildsystem
+
+As of writing this, I'm not entirely sure if JDK 17 is required in order for Android Command Line Tools to function properly.
+I have not tested without it.  To be safe, it is included in the documentation for now.
+
+Now on to the fun part of actually compiling gdext for Android! :)
```

Comment:
Avoid "I" in the tutorial, the book has many authors/contributors.

I assume you used JDK 17 because it's mentioned in the Godot docs?

More neutrally, the above could be written as:

> It's not entirely clear if JDK 17 is required for Android Command Line Tools to function properly; it might be possible without it.

Please also avoid emojis/emoticons and sentences like "Now on to the fun part of actually compiling gdext for Android! :)". Instead, this could be a h2 header.